src/top-bar/org-unit-select/org-unit-select.js

```diff
@@ -31,46 +32,53 @@ const OrgUnitSelect = () => {
     }
     const selectedOrgUnitPath = orgUnit?.path ? [orgUnit.path] : undefined
 
+    useEffect(() => {
```

Comment:
This `useEffect` is required as `onChildrenLoaded` is not called for org unit roots.

Reply:
I would probably consider this to be a bug in the org-unit tree.